docs/manual/developer_guide.adoc

```diff
@@ -555,8 +555,37 @@ Contributions can be made for rules, checks, remediations or even utilities. The
 
 ==== Rules
 
-...
-// shared/references/cce-rhel-avail.txt
+Rules are input in a quasi-XCCDF format, which is basically an XML container.
+They describe the desired state of the system and may contain labels if they are parts of higher-level standards.
```

Comment:
I know what you are getting at but maybe make it more explicit. Talk about human readable text, references, identifiers. Provide examples of higher-level standards to make it clearer what you mean.